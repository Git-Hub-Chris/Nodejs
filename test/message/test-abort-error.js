'use strict';
require('../common');
const { setImmediate } = require('node:timers/promises');

Error.stackTraceLimit = 2;
setImmediate(undefined, { signal: AbortSignal.abort() }).catch(console.error);
