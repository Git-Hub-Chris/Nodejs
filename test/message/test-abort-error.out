AbortError: The operation was aborted
    at setImmediate (node:timers/promises:*:*)
    at Object.<anonymous> (*) {
  code: 'ABORT_ERR',
  [cause]: [DOMException [AbortError]: This operation was aborted]
}
